ktor-server/ktor-server-core/jvm/src/io/ktor/server/engine/EmbeddedServerJvm.kt

```diff
@@ -337,7 +337,7 @@ actual constructor(
             engine.resolvedConnectors().forEach {
                 val host = escapeHostname(it.host)
                 environment.log.info(
-                    "Responding at ${it.type.name.lowercase()}:$it"
+                    "Responding at ${it.type.name.lowercase()}://$host:${it.port}"
```

Comment:
I think it was made to print the correct message for unix-socket connections. Should we properly implement toString instead?